### PR TITLE
[feature] Authorization with google

### PR DESCRIPTION
**Pull request by bastej**
- added possibility to log in into app through google
- tests written

### CODE REVIEW

File: redux/auth/saga.test.ts [hunk 1]
```diff
@@ -194,5 +199,71 @@ describe(`${DOMAIN_NAME}/saga`, () => {
 
       expect(generator.next().done).toBe(true);
     });
+
+    describe('loginWithGoogleSaga', () => {
```
Comment (k-bialucha) [inline redux/auth/saga.test.ts:203]: This nesting is somehow off :smile: this `describe('loginWIthGoogleSaga', ...)` is inside `describe('loginWithFacebookSaga', ...)`. So you should just move it one level up.

File: redux/auth/__snapshots__/saga.test.ts.snap [hunk 2]
```diff
@@ -132,6 +132,64 @@ Object {
 }
 `;
 
+exports[`auth/saga loginWithFacebookSaga loginWithGoogleSaga handles successful signup through google 1`] = `
```
Comment (k-bialucha) [inline redux/auth/__snapshots__/saga.test.ts.snap:135]: Here you can see example of the nested `describe`. Should be fixed after change.

File: redux/auth/saga.ts [hunk 3]
```diff
@@ -1,8 +1,9 @@
 import * as Facebook from 'expo-facebook';
+import * as Google from 'expo-google-app-auth';
 import decode from 'jwt-decode';
 import { call, delay, put, takeLatest } from 'redux-saga/effects';
 
-import { FACEBOOK_APP_ID } from '../../env';
+import { ANDROID_CLIENT_ID, FACEBOOK_APP_ID, IOS_CLIENT_ID } from '../../env';
```
Comment (k-bialucha) [inline redux/auth/saga.ts:6]: New ENV variables are not documented yet. Please add them in our `.env.example.ts` file

File: redux/auth/saga.test.ts (hunk 1 above)
Comment (bastej) [reply redux/auth/saga.test.ts:203]: Really, how did you found it? :smile: thx!

File: redux/auth/saga.ts (hunk 3 above)
Comment (bastej) [reply redux/auth/saga.ts:6]: OK, changes has been pushed. I'm waiting for approval :smile:

**Comment by bastej** [comment]
> I found 2 issues for now: nested `describe` in tests and lacking env documentation. The rest seems to be fine!

OK, changes has been pushed. I'm waiting for approval :smile:

**Pull request merged**